CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # CHANGELOG
+### [v4.14 _(Oct 21, 2021)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.14)
+
```

Comment:
We also send email when order's status change from on-hold to processing in this version. Will we add that to change log as well ?

Reply:
And that mail only for merchant.